common/persistence/cassandraVisibilityPersistence.go

```diff
@@ -156,6 +156,7 @@ func (v *cassandraVisibilityPersistence) RecordWorkflowExecutionStarted(
 		*request.Execution.RunId,
 		common.UnixNanoToCQLTimestamp(request.StartTimestamp),
 		request.WorkflowTypeName,
+		request.RetentionSeconds,
```

Comment:
retention has a different meaning on the server side, to avoid confusion, how about we just call this WorkflowTimeoutSeconds  ? retention usually refers to the retention period for WFHistory on the server side

Reply:
Makes sense.